### ISSUE TITLE
Convert all uses of ansible variables to match new schema

### ISSUE THREAD

**Issue by davidewatson** [samsung-cnct/k2#259]
# Task

The new config schema contains several changes to the organization of stanzas within the config file. The existing ansible tasks must be modified so that variable references match the new configuration file.

This work will be done in the k2recon branch to avoid breaking existing customers while we finalize this change.

**Comment by coffeepac** [comment]
this is blocked by #258

**Comment by davidewatson** [comment]
We should be able to consolidate all assembler tasks into one:

```
diff --git a/ansible/roles/kraken.assembler/tasks/main.yaml b/ansible/roles/kraken.assembler/tasks/main.yaml
index b1c29ef..a55fc69 100644
--- a/ansible/roles/kraken.assembler/tasks/main.yaml
+++ b/ansible/roles/kraken.assembler/tasks/main.yaml
@@ -1,9 +1,14 @@
 ---
 - name: Make sure generated folder for full cloud configs is there
   file: >
-    path="{{ config_base | expanduser }}/{{ kraken_config.cluster }}/cloud-config"
+    path="{{ config_base | expanduser }}/{{ item.name }}/cloud-config"
     state=directory
+  with_items:
+    - "{{ kraken_config.clusters }}"
 
-- include: master.yaml
-- include: node.yaml
-- include: etcd.yaml
\ No newline at end of file
+- include: cloud-config.yaml
+  with_subelements:
+    - "{{ kraken_config.clusters }}"
+    - nodePools
+  loop_control:
+    loop_var: cluster_node_tuple
```

To do this we will have to change most templating to remove the etcd, master, and node file prefix. Since everyone is working on k2recon tickets ATM, I suggest this be done at the end, before the merge into master, but after everyone has checked in their changes.

**Comment by yenicapotediaz** [comment]
Worked on a piece of this ticket, here's my PR for the kraken.config/main.yaml portion: https://github.com/samsung-cnct/k2/pull/340

**Comment by yenicapotediaz** [comment]
Here's my PR https://github.com/samsung-cnct/k2/pull/362

**Comment by coffeepac** [comment]
PR was merged
